config/pipeline.yaml

```diff
@@ -1012,6 +1012,15 @@ jobs:
       tst_cmd: 'svsematest'
     kcidb_test_suite: rt-tests.svsematest
 
+  kselftest-landlock:
+    template: kselftest.jinja2
```

Comment:
This job template no longer exists. It was replaced by [`generic.jinja2`](https://github.com/kernelci/kernelci-pipeline/blob/main/config/runtime/generic.jinja2) - see [the example](https://github.com/kernelci/kernelci-pipeline/blob/main/doc/developer-documentation.md#an-example-of-enabling-a-new-job) for more details.